### PR TITLE
Practice: IPMs

### PR DESCRIPTION
**Pull request by joemoore**

### CODE REVIEW

File: content/practices/ipm/index.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+| **Participants** | • Product Owner<br> • Engineers | • Core team |
+{{</ table >}}
+
+### Sample Agenda & Prompts
+1. Start by framing the conversation for the meeting. Help to set overall project context as needed by briefly recapping recently accepted and in-progress stories or the current state of the newest capabilities in your acceptance environment, to get everyone aligned on where things currently stand as you enter this next iteration. Review the arc of new user stories by outlining what they will add to the product. Present user interface mockups, if available.
+
+
+1. Read through the first user story to explain the business value, the user value, and the acceptance criteria. Allow participants to ask questions, providing clarifications as needed.
+   
+   {{< callout >}}
```
Comment (zmb3) [inline content/practices/ipm/index.md:46]: This feels like excessive use of a callout. If you call out too many things you risk them losing their purpose to emphasize something.

I might recommend a simple (ordered or unordered) list for all of these tips.

Comment (joemoore) [reply content/practices/ipm/index.md:46]: I 100% agree. We have an internal thread about this. Publicly: as part of the [User Research Intro Session](https://tanzu.vmware.com/developer/practices/user-research-introduction-session/), we published a "Dos and Don'ts" companion article; we might be able to do a similar thing for an estimation guide or such.

Comment (joemoore) [reply content/practices/ipm/index.md:35]: Moved the Estimation tips to their own section under "facilitator tips"

File: content/practices/ipm/index.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+- A product backlog
+- A prepared set of prioritized user stories to be reviewed and potentially promoted to the product backlog
+
+# If this practice or workshop has a Miro template: remote: true
+remote: false
+miro_template_url: "URL for related Miro template" 
+
+---
+## How to Use this Method
+### Sprint Planning vs. Iteration Planning
```
Comment (chliuPivotal) [inline content/practices/ipm/index.md:27]: Looks like this header line should be moved down to Line 85 (under "Related Practices" and above the handy table comparing Sprints vs. Iterations).

Comment (joemoore) [reply content/practices/ipm/index.md:27]: Good catch!

Comment (joemoore) [reply content/practices/ipm/index.md]: Fixed

**Comment by joemoore** [comment]
@BrianMMcClain ready to go

**Comment by joemoore** [comment]
OOps hit the wrong button... feel free to review @BrianMMcClain

**Review by BrianMMcClain — APPROVED** [review]

**Pull request merged**